.gitignore

```diff
@@ -0,0 +1,6 @@
+# MAC Files
+.DS_Store
+
+# Serverless Folders
```

Comment:
I'm wondering if we at all need these for anything? @nicolasabousawan

Reply:
every time you deploy, a .serverless file is generated in the server folder. That is not something we want to pull into GitHub. We could probably get rid of ".serverless" since "**/.serverless/**" is a catch all

Reply:
ok that makes more sense, thanks for clarifying!